### PR TITLE
Slash Command Update Syrupy Regression Data

### PR DESCRIPTION
**Pull request by atharva-2001**
### :pencil: Description

**Type:** :vertical_traffic_light: `testing` :roller_coaster: `infrastructure`

Slash command to update regression/syrupy data as their is for refdata

### :pushpin: Resources

Examples, notebooks, and links to useful references.
Please merge https://github.com/tardis-sn/tardis/pull/2413 before this.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: .github/workflows/dispatcher.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   dispatch:
     runs-on: ubuntu-latest
     steps:
-      - name: Slash Command Dispatcher
+      - name: Slash Command Dispatcher Update Refdata
```
Comment (andrewfullard) [inline .github/workflows/dispatcher.yml:10]: Update regression data

**Comment by andrewfullard** [comment]
Where does the regression data go right now? I notice this workflow runs both refdata and regression data. Is this worth merging before we have the syrupy tests in?

File: .github/workflows/dispatcher.yml (hunk 1 above)
Comment (atharva-2001) [reply .github/workflows/dispatcher.yml]: Fixed in https://github.com/tardis-sn/tardis/pull/2423/commits/69b2054d54adc70bab49cc596cf3b316ebb3c52d

**Comment by andrewfullard** [comment]
Slight issue- doesn't this PR rely on a lockfile that includes `syrupy`?

**Review by andrewfullard — APPROVED** [review]

**Review by andrewfullard — APPROVED** [review]

**Pull request closed without merging**